robot/client/client.go

```diff
@@ -330,9 +330,10 @@ func (rc *RobotClient) connect(ctx context.Context) error {
 	if err := rc.connectWithLock(ctx); err != nil {
 		return err
 	}
-
+	rc.Logger().CInfow(ctx, "successfully reconnected to remote at address", "address", rc.address)
 	if rc.notifyParent != nil {
 		rc.notifyParent()
+		rc.Logger().CInfow(ctx, "successfully notified parent after reconnection", "address", rc.address)
```

Comment:
I sense this "parent" terminology will be confusing for most users. When exactly does this get logged? When an RDK remote connection or reconnection successfully sends to the `triggerConfig` channel or encounters a `closeContext` error? I would vote to not have this log (or have it at the debug level) unless you feel that connecting + being unable to notify parent is still a situation that exists after these changes.

Reply:
updated to `rc.Logger().CDebugw(ctx, "successfully notified parent after (re)connection", "address", rc.address)`